src/open_inwoner/components/templatetags/side_navigation.py

```diff
@@ -0,0 +1,144 @@
+import contextlib
+import logging
+
+from django import template
+
+from menus.menu_pool import menu_pool
+
+register = template.Library()
+logger = logging.getLogger(__name__)
+
+
+@register.simple_tag(takes_context=True)
+def react_sidenav_data(context):
+    """Template tag to provide menu data for React Sidenav component"""
+    request = context["request"]
+
+    try:
+        # Get the menu renderer
+        renderer = menu_pool.get_renderer(request)
+
+        # Get all menu nodes
+        all_nodes = renderer.get_nodes()
+        logger.debug(f"Total nodes found: {len(all_nodes)}")
+
+        # Find the "home" node - users are required to set this ID!
+        # else we need to exclude all the other pages from this menu node
+        home_node = None
+        for node in all_nodes:
+            # Check if this is the home page
+            if (
+                (hasattr(node, "id") and node.id == "home")
+                or (hasattr(node, "reverse_id") and node.reverse_id == "home")
+                or (
+                    getattr(node, "title", "").lower() == "overzicht"
+                    and node.get_absolute_url() == "/"
+                )
+            ):
+                home_node = node
+                logger.debug(f"Found home node: {node.title}")
+                break
```

Comment:
I think we should stick to one explicit condition for locating the `home` node, to make the behavior more predictable (and issues less ambiguous to fix):

```suggestion
            node_attr = getattr(node, "attr", {})
            if node_attr.get("reverse_id", None) == "home"):
                home_node = node
                logger.debug(f"Found home node: {node.title}")
                break
```